src/rep.rs

```diff
@@ -460,8 +508,11 @@ pub struct History {
     #[serde(deserialize_with = "datetime_from_unix_timestamp")]
     pub created: DateTime<Utc>,
     #[cfg(not(feature = "chrono"))]
-    pub created: u64,
+    pub created: i64,
```

Comment:
Looking at the codebase there are 85 occurances of `u64` and only 3 of `i64` so I'd stick with that.

Reply:
I was guided by the [API documentation](https://docs.docker.com/engine/api/v1.41/#operation/ImageHistory): `integer <int64>`.

If we look at the codebase we can find other flaws as well: missing fields, don't respecting required/optional fields. I think it is better to write the code according to API documentation.

Reply:
Well, agreed. Perhaps it's better to go the other way and start being more strict about the type.

Reply:
So I'm just trying to make sure to perhaps cleanup my PRs. Do you think that by `integer` they mean a `u64` and by `integer <int64>` it means `i64`?